README.md

```diff
@@ -1 +1,64 @@
+http://localhost:8080/swagger-ui/index.html#/ you will have this functionality.
+
+# Controllers functionality
+Authentication management allows you to register in the service system. After registering users have the ability to log in and have access to
+the endpoints listed below.
+
+![](C:\Users\barma\IdeaProjects\book-store\images\Authentication_management_2.jpg)
+
+After passing authentication you have functionality listed below:
+
+In the Shopping Cart section you have access to your previously formed shopping cart, if you haven't one there will be your empty shopping cart.
+You can add books to your shopping cart, change their quantity or delete books from the shopping cart.
+
+![](C:\Users\barma\IdeaProjects\book-store\images\ShoppingCart_management_2.jpg)
+
+In Order management you can get a list of all previously placed orders, you can see items in certain order from that list,
+you can also see detailed information about that item. You can place a new order. Users with role admin have the ability to change
+status of the orders.
+
+![](C:\Users\barma\IdeaProjects\book-store\images\Order_management_2.jpg)
+
+In the Books section you have the ability to see all the books available at the moment, watch detailed information
+about the book chosen or search for the book. The ability to add, update or delete books is available only for users with role ADMIN.
+
+![](C:\Users\barma\IdeaProjects\book-store\images\Book_management_2.jpg)
+
+In the Category sections you can see all the categories/genres of books available, you can peek at one and see for detailed information
+or to see all the books that are present in this particular category.  The ability to add, update or delete new categories is available
+only for users with the role ADMIN.
+
+![](C:\Users\barma\IdeaProjects\book-store\images\Categories_management_2.jpg)
+    
+# Possible usage <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\docker logo.png"/> <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\aws-logo.png"/>
+One of the possible ways to use this book-service is to make an image of this part of the application.
+After that image can be pushed to one of the cloud services for example AWS. Database connection will be configured there.
+This will be the back-end part of the web app. Just like that there can be configured front-end parts.
+Different parts of the application communicate between each other through the http protocol.
+
+![](C:\Users\barma\IdeaProjects\book-store\images\architecture.jpg)
+
+It could be as stand-alone application consists of front-end, back-end parts only, or it could be used as microservice, small part of
+more complicated ecosystem where book-store service will be responsible only for book selling. But this will have common security
+and some other settings with other microservices.
+
+![](C:\Users\barma\IdeaProjects\book-store\images\Untitled-3.jpg)
+
+ 
```

Comment:
1. How to run your application? - didn't find about this

2. Add a Loom video (2-4 min) where you show how your project works

3. Not all technologies included 

4. You can make your readme more engaging with the different capabilities of .md. 
Make more topics, group your texts, add code examples (e.g. how to run your app), notations, order/unordered lists